lua/wire/wireshared.lua

```diff
@@ -1030,9 +1062,9 @@ hook.Add("InitPostEntity","WireForceLimit",function()
 	min_force = -max_force
 end)
 
-
 -- Nan never equals itself, so if the value doesn't equal itself replace it with 0.
 function WireLib.clampForce( v )
+	v = Vector(v[1], v[2], v[3])
```

Comment:
I realize that vectors in E2 are tables (mostly) but is `v[1]` the same thing as `v.x` if the vector is actually a vector? Maybe we need type checks here? Testing required

Reply:
It is.